packages/aws-cdk/lib/init-templates/lib/typescript/jest.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
```

Comment:
You still have to specify `testMatch: [ '**/*.test.ts']` because after `npm run build` you will end up with `.js` files in your test folder.

I would also add `testEnvironment: 'node'` + may be coverage ?

Reply:
Added the testMatch. I don't think we need to get into adding test coverage for the initial templates.